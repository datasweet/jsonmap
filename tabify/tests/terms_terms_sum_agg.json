{
    "took": 4,
    "timed_out": false,
    "_shards": {
        "total": 5,
        "successful": 5,
        "skipped": 0,
        "failed": 0
    },
    "hits": {
        "total": 111396,
        "max_score": 0,
        "hits": []
    },
    "aggregations": {
        "2": {
            "doc_count_error_upper_bound": -1,
            "sum_other_doc_count": 105158,
            "buckets": [
                {
                    "1": {
                        "value": 94547
                    },
                    "3": {
                        "doc_count_error_upper_bound": 0,
                        "sum_other_doc_count": 0,
                        "buckets": [
                            {
                                "1": {
                                    "value": 94547
                                },
                                "key": "Hamlet",
                                "doc_count": 1582
                            }
                        ]
                    },
                    "key": "HAMLET",
                    "doc_count": 1582
                },
                {
                    "1": {
                        "value": 76899
                    },
                    "3": {
                        "doc_count_error_upper_bound": 0,
                        "sum_other_doc_count": 0,
                        "buckets": [
                            {
                                "1": {
                                    "value": 76899
                                },
                                "key": "Othello",
                                "doc_count": 1161
                            }
                        ]
                    },
                    "key": "IAGO",
                    "doc_count": 1161
                },
                {
                    "1": {
                        "value": 71866
                    },
                    "3": {
                        "doc_count_error_upper_bound": 0,
                        "sum_other_doc_count": 0,
                        "buckets": [
                            {
                                "1": {
                                    "value": 71866
                                },
                                "key": "Loves Labours Lost",
                                "doc_count": 647
                            }
                        ]
                    },
                    "key": "BIRON",
                    "doc_count": 647
                },
                {
                    "1": {
                        "value": 62945
                    },
                    "3": {
                        "doc_count_error_upper_bound": 0,
                        "sum_other_doc_count": 0,
                        "buckets": [
                            {
                                "1": {
                                    "value": 62945
                                },
                                "key": "Othello",
                                "doc_count": 928
                            }
                        ]
                    },
                    "key": "OTHELLO",
                    "doc_count": 928
                },
                {
                    "1": {
                        "value": 60243
                    },
                    "3": {
                        "doc_count_error_upper_bound": 0,
                        "sum_other_doc_count": 5,
                        "buckets": [
                            {
                                "1": {
                                    "value": 28740
                                },
                                "key": "Richard III",
                                "doc_count": 743
                            },
                            {
                                "1": {
                                    "value": 9809
                                },
                                "key": "King Lear",
                                "doc_count": 364
                            },
                            {
                                "1": {
                                    "value": 9238
                                },
                                "key": "Henry VI Part 3",
                                "doc_count": 266
                            },
                            {
                                "1": {
                                    "value": 9156
                                },
                                "key": "Henry VI Part 2",
                                "doc_count": 320
                            },
                            {
                                "1": {
                                    "value": 3161
                                },
                                "key": "Henry VI Part 1",
                                "doc_count": 222
                            }
                        ]
                    },
                    "key": "GLOUCESTER",
                    "doc_count": 1920
                }
            ]
        }
    },
    "status": 200
}
