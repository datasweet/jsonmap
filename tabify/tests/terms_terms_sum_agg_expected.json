[
  { "2": "HAMLET", "2#doc_count": 1582, "2#1": 94547, "2#3": "Hamlet", "2#3#doc_count": 1582, "2#3#1": 94547 },
  { "2": "IAGO", "2#doc_count": 1161, "2#1": 76899, "2#3": "Othello", "2#3#doc_count": 1161, "2#3#1": 76899 },
  { "2": "BIRON", "2#doc_count": 647, "2#1": 71866, "2#3": "Loves Labours Lost", "2#3#doc_count": 647, "2#3#1": 71866 },
  { "2": "OTHELLO", "2#doc_count": 928, "2#1": 62945, "2#3": "Othello", "2#3#doc_count": 928, "2#3#1": 62945 },
  { "2": "GLOUCESTER", "2#doc_count": 1920, "2#1": 60243, "2#3": "Richard III", "2#3#doc_count": 743, "2#3#1": 28740 },
  { "2": "GLOUCESTER", "2#doc_count": 1920, "2#1": 60243, "2#3": "King Lear", "2#3#doc_count": 364, "2#3#1": 9809 },
  { "2": "GLOUCESTER", "2#doc_count": 1920, "2#1": 60243, "2#3": "Henry VI Part 3", "2#3#doc_count": 266, "2#3#1": 9238 },
  { "2": "GLOUCESTER", "2#doc_count": 1920, "2#1": 60243, "2#3": "Henry VI Part 2", "2#3#doc_count": 320, "2#3#1": 9156 },
  { "2": "GLOUCESTER", "2#doc_count": 1920, "2#1": 60243, "2#3": "Henry VI Part 1", "2#3#doc_count": 222, "2#3#1": 3161 }
]
